package/src/components/Channel/Channel.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { PropsWithChildren, useCallback, useEffect, useMemo, useRef, useState } from 'react';
-import { KeyboardAvoidingViewProps, StyleSheet, Text, View } from 'react-native';
+import {
+  FlatList as FlatListDefault,
```

Comment:
```suggestion
  FlatList,
```

this should be possible now right?

Reply:
No, this is done to assign the name to the prop FlatList below which will be conflicting if same name used.